backend/models/shop_deployment.js

```diff
@@ -0,0 +1,21 @@
+module.exports = (sequelize, DataTypes) => {
+  const ShopDeployment = sequelize.define(
+    'ShopDeployment',
+    {
+      shopId: DataTypes.INTEGER,
+      domain: DataTypes.STRING,
+      ipfsGateway: DataTypes.STRING,
+      ipfsHash: DataTypes.STRING
```

Comment:
Might want to add created/updated so they can be used?

Reply:
I don't think you need to declare those explicitly in the model. Sequelize does it for you automagically :) https://sequelize.org/v5/manual/models-definition.html#timestamps